modules/rdf_draft/rdf_draft.module

```
@@ -67,25 +42,25 @@ function rdf_draft_entity_type_alter(array &$entity_types) {
  */
 function rdf_draft_entity_operation(EntityInterface $entity) {
   $operations = [];
-  $storage = \Drupal::service('entity.manager')->getStorage($entity->getEntityTypeId());
-  if ($storage  instanceof RdfEntitySparqlStorage) {
-    $definitions = $storage->getGraphDefinitions();
-    foreach ($definitions as $name => $definition) {
-      if ($entity->hasLinkTemplate('rdf-draft-' . $name) && $entity->hasGraph($name)) {
+  try {
+    /** @var \Drupal\rdf_entity\RdfEntitySparqlStorageInterface $storage */
+    $storage = \Drupal::entityTypeManager()->getStorage($entity->getEntityTypeId());
+  }
+  catch (\Exception $exception) {
+    return [];
```

Is masking the exception intended here?